.gitignore

```diff
@@ -15,3 +15,6 @@ jspm_packages
 /coverage
 /.tmp
 .nyc_output
+
+# Ignore IDE files
+/.idea
```

Comment:
good stuff - I just add this in my computer global gitignore - i suggest you do the same